translations/i18n.js

```diff
@@ -0,0 +1,24 @@
+const path = require("path")
+const electron = require('electron')
+const fs = require('fs');
+let loadedLanguage;
+let app = electron.app ? electron.app : electron.remote.app
+
+module.exports = i18n;
+
+function i18n() {
+    if(fs.existsSync(path.join(__dirname, app.getLocale() + '.json'))) {
+         loadedLanguage = JSON.parse(fs.readFileSync(path.join(__dirname, app.getLocale() + '.json'), 'utf8'))
```

Comment:
polish: let's put it in separate function `JSON.parse(fs.readFileSync(path.join(__dirname, localeName), 'utf8'))` to avoid code duplication?